### DISCUSSION TITLE
After upgrade to Frigate 14 - Frozen Cameras

### DISCUSSION THREAD

**Discussion by juampick** [blakeblackshear/frigate#13371] [General]
Hi, After upgrading (yesterday) to Frigate 14, things started to work badly on Frigate. 

On dashboard live view, some cameras are frozen. I restarted Frigate, and works fine for some minutes, and then it gets frozen again. Before everything was working fine!

**Comment by hawkeye217** [comment]
Be sure to read the [release notes](https://github.com/blakeblackshear/frigate/releases/tag/v0.14.0) and the [FAQ](https://github.com/blakeblackshear/frigate/discussions/12724), specifically Usage question 5.

If you do believe there is a problem, please open a support request with all of the required information and we'd be happy to take a look.
